ui/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:lts-alpine
+
+# install simple http server for serving static content
+RUN npm install -g http-server
+
+# make the 'app' folder the current working directory
+WORKDIR /app
+
+# copy both 'package.json' and 'package-lock.json' (if available)
+COPY package*.json ./
+
+# install project dependencies
+RUN npm install
```

Comment:
I'm not sure about this, but wouldn't it be more logical to copy files and then install? Instead of installling and then copy the files?

Reply:
Yes

Also because I can't figure out how to commend my own code, I will try to avoid a global npm install for http-server, so that the build steps and entrypoint are more transparent.

Doing this and above recs! Danke!

Reply:
Switching order worked, therefore done.